src/parse/decl.rs

```diff
@@ -947,6 +947,14 @@ impl<I: Iterator<Item = Lexeme>> Parser<I> {
                 if let Type::Array(to, _) = ctype {
                     ctype = Type::Pointer(to);
                 }
+
+                // Fix #142 (C11 Standard 6.7.6.3 paragraph 8)
```

Comment:
Normally I put a link to the issue in the test case, not in the code, unless it's something really hacky. The link to the standard is good, though.

Reply:
Sounds good. I'll change to use `is_function()` while I'm making this change anyway.